utils/parser.py

```diff
@@ -73,6 +77,18 @@ async def filter_and_sort_streams(
             ):
                 filtered_reasons["Requires Private Tracker Support"] += 1
                 continue
+        # It's a series if season and episode is passed in
```

Comment:
lets not use the parser function to filter episode. I want to create a migration script to clean up the data in db `source` other than "Contribution Stream"